printer_mods/Triano/Ender_3Pro_Switchwire/README.md

```diff
@@ -0,0 +1,97 @@
+#  Ender 3 Pro and V2 to Switchwire Mod
+
+This is a conversion of the Ender 3 Pro and V2 to a Switchwire type coreXZ printer.  My goal was to use most of the Ender 3 components while maintaining the core Voron design look and feel.
+
+![Home](Images/SN049.jpg)
+![Home](Images/Home.png)
+![Home](Images/Bottom.png)
+![Home](Images/Top.png)
+![Home](Images/Back.png)
+![Home](Images/Front.png)
+![Home](Images/Left.png)
+![Home](Images/Right.png)
+
+## Overview
+- This mod uses most of the key components from the Ender 3 Pro (E3P) while providing a coreXZ capability.  The highlights of this include:
+1. The frame is all E3P.
+2. The electronics are the stock Switchwire ones, but the E3P 24V power supply is used.  I wanted to get all the electronics under the deck as this follows the Switchwire design.  I had to get a little creative with the mounts to make this work.  I did not include the LCD display.   Mainsail (MainsailOS) is so amazing, I never use the LCD on the Voron 2.4 so I left it out.  It can easily be added.
+3. The deck is easily printed in 4 sections.
+4. The XZ and gantry are slightly modified from the Switchire baseline.  The X asis rail is 300mm and is a tight squeeze on the shortened 2020 extrusion.  I wanted to keep the standard rail length in case I wanted to build a stock Switchwire.
```

Comment:
Missed this last time, change `Switchire` to `Switchwire`.

Reply:
CAD files spaces fixes and typo fixed.